src/components/HarvestersDialog/index.css

```diff
@@ -0,0 +1,3 @@
+#harvesters-dialog .Dialog__content {
+    width: 700px;
+  }
```

Comment:
Indentation is a little off. Don’t you have Prettier installed?
```suggestion
#harvesters-dialog .Dialog__content {
    width: 700px;
  }
```